flox-bash/tests/multi-env.bats

```diff
@@ -0,0 +1,66 @@
+#! /usr/bin/env bats
+# -*- mode: bats; -*-
+# ============================================================================ #
+#
+# Test the `FLOX_ENV' variable set by `flox activate' is set appropriately when
+# multiple environments are indicated on the CLI.
+#
+# ---------------------------------------------------------------------------- #
+
+bats_load_library bats-support;
+bats_load_library bats-assert;
+bats_require_minimum_version 1.5.0;
+
```

Comment:
```suggestion
```

Reply:
Does it work without those? Both `integration.bats` and `package.bats` have these blocks as well.

Reply:
Yeah it passed, I think because of the load test_support